routes/examples/paginationExample.js

```diff
@@ -0,0 +1,74 @@
+const router = require("express").Router();
+const Product = require("../../models/ProductModel");
+const authenticateUser = require("../verifyToken");
+
+router.get("/get-products", async (req, res) => {
```

Comment:
We don't need to set routes as get-products, create-product, update-product or delete-product. It can simply be `/products` and the POST, PUT, GET and DELETE methods will justify the functionality of that route.